### ISSUE TITLE
Using ProjectGenerator to create an iOS ofxPd project?

### ISSUE THREAD

**Issue by batchku** [danomatika/ofxPd#78]
I'd like to build the `pdExampleIOS` project for iOS.  When I use the ProjectGeneator to import this project, the resulting XCode project is not an iOS project, but rather an MacOS project.  What do I need to do to change that? Are there other instructions for how to get the `pdExampleIOS` project 
 built and onto an iOS device?

Thank you!

**Comment by danomatika** [comment]
If what you meant was "the resulting XCode project is not an iOS project, but rather an **macOS** project", the answer is that you should either use the PG included with the OF iOS bundle and/or enable the advanced PG options and manually select the platform.

**Comment by batchku** [comment]
Thanks @danomatika ; I had never used the that option in PG; I see it now.
I notice three things:

1. I "import" the `pdExampleIOS` and generate a project; PG does recognize that the `ofxPd` addon is required; however, when i try to build the project, XCode is not able to locate `ofxPd.h`
2. I added the `ofxPd` packages `src` folder to the XCode header search path; indicated recursive; still can't find it
3.  the `addons` folder that is in the Xcode project, generated by PG is empty; I tried adding the `ofxPd` folder to that `addons` folder by dragging it and adding to the targets; it still can't find it


I must still be doing something wrong in the way I am generating my project with PG.  What's my problem?

Thanks in advance.
